automatic/k9s/update.ps1

```diff
@@ -34,7 +34,11 @@ function global:au_GetLatest {
 
   $checksumAsset = $domain + ($download_page.Links | ? href -match "checksums\.txt$" | select -first 1 -expand href)
   $checksum_page = Invoke-WebRequest -Uri $checksumAsset -UseBasicParsing
-  $filename64 = -join('k9s_v', $version, '_Windows_x86_64.tar.gz')
+  if($VersionStyle -eq 0) {
+    $filename64 = 'k9s_Windows_x86_64.tar.gz'
+  } elseif ($VersionStyle -eq 1) {
+    $filename64 = -join('k9s_v', $version, '_Windows_x86_64.tar.gz')
+  }
   $checksum64 = [regex]::Match($checksum_page, "([a-f\d]+)\s*$([regex]::Escape($filename64))").Groups[1].Value
```

Comment:
To refer to your question about what I mean, I was thinking something along the lines of:

```suggestion
  $regexFormat = "([a-f\d]+)\s*{0}"
  $possibleFileNames = @(
    'k9s_Windows_x64_64.tar.gz'
    ('k9s_v{0}_Windows_x86_64.tar.gz' -f $version)
  )
  foreach ($fileName64 in $possibleFileNames) {
    $regex = $regexFormat -f ([regex]::Escape($fileName64))
    if ($checksum_page -match $regex) {
      $checksum64 = $Matches[1]
      break
    }
  }
```

**NOTE: I have not tested this snippet if it works**